src/vscode-dts/vscode.proposed.debugSelectedConfiguration.d.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	export interface SelectedDebugConfiguration {
+		name: string;
+	}
+
+	export namespace debug {
+		/**
+		 * The debug configuration, which is currently selected by the user.
+		 */
+		export const selectedConfiguration: SelectedDebugConfiguration | undefined;
```

Comment:
This is object and not a string, so additional information can be added in the future, if needed. Internally we store only name, but again, we can change internal API but not public.